PINCE.py

```diff
@@ -2932,6 +2932,7 @@ class MemoryViewWindowForm(QMainWindow, MemoryViewWindow):
     show_memory_view_on_stop: bool = False
     instructions_per_scroll: int = 3
     bytes_per_scroll: int = 0x40
+    stack_from_base_pointer = False
```

Comment:
You mean change it to `self.stack_from_base_pointer` at line 2935?

Reply:
Yes, `self.stack_from_base_pointer: bool = False`

Reply:
This doesn't seem to work. Only `stack_from_base_pointer: bool = False`. Did you mean somewhere else?

Reply:
In __init__ I meant, that’s where instance variables go.

On further inspection, I see that those are class variables because they act like settings, @korcankaraokcu I think we should note to move the others to init if we already have those in settings, otherwise keep them here.

@OpenNetSurfer leave the line like this since we don’t have bp/sp in settings. Just add the bool type hint